src/itest/resources/features/insolvency-data.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Process insolvency delta information
+
+  Scenario: Processing insolvency delta information successfully
+
+    Given Insolvency delta consumer service is running
+    When a message is published to the topic "insolvency-delta" and consumed
+    Then verify PUT method is called on insolvency-data-api service
```

Comment:
Can we please add data table to parameterise different input/output files, I believe QA team provided various input files for this integration story instead of one payload?